### PR TITLE
[DEV-3412] [DEV-3414] [DEV-3424] [DEV-3426] DTO changes

### PR DESCRIPTION
**Pull request by lapatric**

### CODE REVIEW

**Comment by lapatric** [comment]
- [ ] Await [Packages update](https://github.com/DFXswiss/packages/pull/86)

File: src/components/exchange-rate.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export function ExchangeRate({
   const minFee = `, min. ${fees.min}${feeSymbol}`;
   const dfxFee = `${fees.dfx}${feeSymbol} (${(fees.rate * 100).toFixed(2)}%${fees.min ? minFee : ''})`;
   const networkFee = `${fees.network}${feeSymbol}`;
+  const bankFee = type !== TransactionType.SWAP && `${fees.bank}${feeSymbol}`;
   const networkStartFee = fees?.networkStart ? `${fees?.networkStart}${feeSymbol}` : undefined;
```
Comment (davidleomay) [inline src/components/exchange-rate.tsx:44]: What if `fees.bank` is null?

File: src/screens/kyc.screen.tsx [hunk 2]
```diff
@@ -566,29 +566,39 @@ function ContactData({ code, mode, isLoading, step, onDone, onBack, showLinkHint
 }
 
 function PersonalData({ rootRef, mode, code, isLoading, step, onDone, onBack }: EditProps): JSX.Element {
-  const { allowedCountries, translate, translateError } = useSettingsContext();
+  const { allowedCountries, allowedOrganizationCountries, translate, translateError } = useSettingsContext();
   const { setPersonalData } = useKyc();
   const { countryCode } = useGeoLocation();
 
+  const [countries, setCountries] = useState<Country[]>([]);
   const [isUpdating, setIsUpdating] = useState(false);
```
Comment (davidleomay) [inline src/screens/kyc.screen.tsx:574]: Can't we just use a simple variable?
`const countries = selectedAccountType === AccountType.PERSONAL ? allowedCountries : allowedOrganizationCountries;`

File: src/components/exchange-rate.tsx (hunk 1 above)
Comment (lapatric) [reply src/components/exchange-rate.tsx:44]: Here in the `ExchangeRate` component, the `fees.bank` comes from the buy, sell and swap `PaymentInfo` where `fees.bank` should in theory never be `null` 👍

File: src/screens/kyc.screen.tsx (hunk 2 above)
Comment (lapatric) [reply src/screens/kyc.screen.tsx:574]: I refactored this part a bit, it still uses the state variable but should be improved, let me know what you think ^^

**Review by davidleomay — APPROVED** [review]

File: src/screens/payment-routes.screen.tsx [hunk 3]
```diff
@@ -810,7 +810,7 @@ function PaymentLinkForm({
       if (prefilledPaymentConfig) {
         reset({
           configStandards: toConfigStandards(prefilledPaymentConfig.standards, prefilledPaymentConfig.blockchains),
-          configMinCompletionStatus: prefilledPaymentConfig.minCompletionStatus,
+          configMinCompletionStatus: prefilledPaymentConfig.minCompletionStatus as any,
           configDisplayQr: prefilledPaymentConfig.displayQr,
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:814]: `as any`?

**Pull request merged**